release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4001,4 +4001,24 @@ $ oc adm release info 4.12.30 --pullspecs
 [id="ocp-4-12-30-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+
+[id="ocp-4-12-31"]
+=== RHBA-2023:4756 - {product-title} 4.12.31 bug fix update
+
+Issued: 2023-08-31
+
+{product-title} release 4.12.31, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.
```

Comment:
```suggestion
{product-title} release 4.12.31  is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4756[RHBA-2023:4756] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4759[RHBA-2023:4759] advisory.
```

Reply:
Well spotted, @opayne1 . I need to pay more attention when it comes to bug fix only advisories. Updated.